src/axom/quest/examples/quest_distributed_distance_query_example.cpp

```diff
@@ -1009,6 +1181,14 @@ int main(int argc, char** argv)
     }
   }
 
+  int errCount = 0;
+  int localErrCount = 0;
+  if(params.checkResults && rankHasQueryPoints)
+  {
+    localErrCount = query_mesh_wrapper.checkClosestPoints<DIM>(circle, params);
+  }
```

Comment:
I think this will take some minor refactoring, because when we don't have query points, we don't set the "cp_index" to -1.  It's initialized down in the lower-level logic.  I'll talk with you about the best way to fix this.

Reply:
Per conversation with @kennyweiss, this suggestion has been dropped.